src/components/lineup/types.ts

```diff
@@ -0,0 +1,108 @@
+import { ID, UID } from 'audius-client/src/common/models/Identifiers'
+import Kind from 'audius-client/src/common/models/Kind'
+import { Lineup as LineupData } from 'audius-client/src/common/models/Lineup'
+import { LineupActions } from 'audius-client/src/common/store/lineup/actions'
+import { Maybe } from 'audius-client/src/common/utils/typeUtils'
+import { SectionListProps } from 'react-native'
+
+export enum LineupVariant {
+  MAIN = 'main',
+  SECTION = 'section',
+  CONDENSED = 'condensed',
+  PLAYLIST = 'playlist'
+}
+
+export type LineupItem =
+  | {
```

Comment:
personally kinda like making these their own types, but maybe hard to name the loaded state?